app/code/BodenkoVV/AskQuestion/Controller/Adminhtml/Question/Edit.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace BodenkoVV\AskQuestion\Controller\Adminhtml\Question;
+
+use Magento\Backend\App\Action;
+
+/**
+ * Class Edit
+ * @package BodenkoVV\AskQuestion\Controller\Adminhtml\Question
+ */
+class Edit extends Action
+{
+    /**
+     * Authorization level of a basic admin session
+     *
+     * @see _isAllowed()
+     */
+    const ADMIN_RESOURCE = 'BodenkoVV_AskQuestion::save';
+
+    /**
+     * Core registry
+     *
+     * @var \Magento\Framework\Registry
+     */
+    protected $_coreRegistry;
+
+    /**
+     * @var \Magento\Framework\View\Result\PageFactory
+     */
+    protected $resultPageFactory;
+
+    /** @var string $_pageName */
+    protected $_pageName='Question Edit Item';
+
+    /**
+     * @param Action\Context $context
+     * @param \Magento\Framework\View\Result\PageFactory $resultPageFactory
+     * @param \Magento\Framework\Registry $registry
+     */
+    public function __construct(
+        Action\Context $context,
+        \Magento\Framework\View\Result\PageFactory $resultPageFactory,
+        \Magento\Framework\Registry $registry
+    ) {
+        $this->resultPageFactory = $resultPageFactory;
+        $this->_coreRegistry = $registry;
+        parent::__construct($context);
+    }
+
+    /**
+     * Init actions
+     *
+     * @return \Magento\Backend\Model\View\Result\Page
+     */
+    protected function _initAction()
+    {
+        // load layout, set active menu and breadcrumbs
+        /** @var \Magento\Backend\Model\View\Result\Page $resultPage */
+
+        $resultPage = $this->resultPageFactory->create();
+        $resultPage->addBreadcrumb(__($this->_pageName), __($this->_pageName));
+        $resultPage->getConfig()->getTitle()->prepend(__($this->_pageName));
+        return $resultPage;
+    }
+
+    /**
+     * Edit CMS page
+     *
+     * @return \Magento\Backend\Model\View\Result\Page|\Magento\Backend\Model\View\Result\Redirect
+     * @SuppressWarnings(PHPMD.NPathComplexity)
+     */
+    public function execute()
+    {
+        // 1. Get ID and create model
+        $id = $this->getRequest()->getParam('id');
+        $model = $this->_objectManager->create(\BodenkoVV\AskQuestion\Model\Question::class);
```

Comment:
Avoid using of object manager. It's possible to inject either QuestionFactory or Question object in the constructor

Reply:
+